### PR TITLE
#165413056 Setup an empty react project

### PR DESCRIPTION
**Pull request by jabichris**
#### Description
This PR is for setting up an empty react project from scratch.

#### Type of change
N/A

#### How Has This Been Tested
After cloning the repo, do git fetch, then check out to `ch-setup-react-165413056` branch. do `npm install` to get all the packages. Then run `npm run start`.

#### Checklist
N/A

#### Pivotal tracker ID
#165413056

### CODE REVIEW

**Review by AJAkimana — COMMENTED** [review]
Good work @jabichris, but you forget to put `@babel/plugin-proposal-class-properties` in `package.json` file. Also, I would suggest putting `port` in environment variables

**Review by ericnyirimana — CHANGES_REQUESTED** [review]
Can you install @babel/plugin-proposal-class-properties and save it into the package.json so that we can start the app. Thank you

**Review by Kabalisa — CHANGES_REQUESTED** [review]
@jabichris good job. in addition to others' feedback, I noticed that the title of our app in ```index.js``` is ```react tutorial```. can you please change it to our app's title ```authors haven```.

**Comment by macphilips** [comment]
It appears task on this PR isn't complete. Where did you initialize the React app?

**Comment by macphilips** [comment]
Also, add a READ.me to the report

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+( ()=> {  console.log("hey Riders");})();
```
Comment (Niyitangasam) [inline src/index.js]: Great work but instead of this console, you can consider initializing react and render something.

Comment (jabichris) [reply src/index.js]: @Niyitangasam, I worked on this issue.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="theme-color" content="#000000">
+    <title>Author Haven</title>
+</head>
+
+<body> <noscript> You need to enable JavaScript to run this app. </noscript>
+    <div id="root"></div>
+    <!--      This HTML file is a template.      If you open it directly in the browser, you will see an empty page.      You can add webfonts, meta tags, or analytics to this file.      The build step will place the bundled scripts into the <body> tag.    -->
+</body>
+
+</html>
```
Comment (ericnyirimana) [inline src/index.html]: Add a space after ```</html>```

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+render(<h1>Hello Riders</h1>,document.getElementById('root'))
+
+
+// ( ()=> {  console.log("hey Riders");})();
```
Comment (ericnyirimana) [inline src/index.js]: Remove this commented console log

**Review by ericnyirimana — APPROVED** [review]

**Review by Kabalisa — APPROVED** [review]

**Pull request merged**